CWE/events/italy.txt

```diff
@@ -455,6 +455,1043 @@ country_event = {
 	
 }
 
+#1953 elections
+country_event = {
+	id = 8020312
+	title = "$YEAR$ Italian General Election"
+	desc = EVTDESC8020312
+	picture = "turkey_election_1950"
+	
+	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1953
+		NOT = { year = 1958 }
+		
+		government = democracy
+		war = no
+		is_vassal = no
+		
+		OR = { has_country_flag = 1948_ITA_Centrism has_country_flag = 1948_ITA_FDP }
+	}
+
+	mean_time_to_happen = { months = 2 }
+
+	option = {
+		name = "Democratic Centre (DC-PSDI-PLI-PRI)"
+		ruling_party_ideology = conservative
+		any_pop = { 
+			ideology = { factor = 0.1 value = conservative }
+			ideology = { factor = 0.05 value = big_tent }
+			ideology = { factor = 0.02 value = liberal }
+		}
+		
+		political_reform = jefferson_method
+		
+		set_country_flag = 1953_ITA_Legge_Truffa
+		ai_chance = { factor = 0.2 }
+	}
+	
+	option = {
+		name = "One Party DC (DC)"
+		ruling_party_ideology = conservative
+		any_pop = { 
+			ideology = { factor = -0.1 value = conservative }
+			ideology = { factor = 0.02 value = nationalist }
+		}
+		any_pop = { 
+			limit = { strata = poor }
+			ideology = { factor = 0.03 value = communist }
+		}
+		any_pop = { 
+			limit = { OR = { strata = poor strata = middle } }
+			ideology = { factor = 0.03 value = socialist }
+		}
+		set_country_flag = 1953_ITA_Centrism
+		ai_chance = { factor = 0.7 }
```

Comment:
Please follow the convention of putting the historical option as the first option in all events and if it is not a historical option, the option with the highest ai_chance should be first. In an event with 3 options, the historical option must have at least 80% ai_chance